Source/ProgressionSystemRuntime/Private/LevelActors/PSStarActor.cpp

```diff
@@ -48,41 +63,89 @@ void APSStarActor::OnLocalCharacterReady(APlayerCharacter* Character, int32 Char
 	}
 }
 
+// Called when the current game state was changed
+void APSStarActor::OnGameStateChanged(ECurrentGameState GameState)
+{
+	if (AMyGameStateBase::Get().GetCurrentGameState() == ECurrentGameState::Menu)
```

Comment:
No need to call `GetCurrentGameState()` here, just use `GameState` function argument

Reply:
done here: https://github.com/h4rdmol/ProgressionSystem/pull/14/commits/9c7ee7a8960dc8cbac8a37fd197a905bc47d19c8

Reply:
@h4rdmol, I mean, just use directly function's argument (green), instead of Getter (red) as it's just a bit cheaper to use already provided parameter
![image](https://github.com/user-attachments/assets/916487c6-75a1-4d7f-9c22-5d9b54ee8fbb)

Reply:
done as part of https://github.com/h4rdmol/ProgressionSystem/pull/14/commits/df58201938adfe61a41317402f8c79a7dd406cce